### PR TITLE
doc: Add readme, contributing.md; update owners & security contacts

### PR DESCRIPTION
**Pull request by swatisehgal**
This PR updates README, adds contribution guidelines and populates security contacts and owners correctly.

Signed-off-by: Swati Sehgal <swsehgal@redhat.com>

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -1,16 +1,11 @@
...
 
 approvers:
-- dims
-- derekwaynecarr
-- dchen1107
-- Huang-Wei
-- sig-node-api-approvers
-- api-approvers
+  - swatisehgal
+  - fromanirh
```
Comment (ffromani) [inline OWNERS]: please use the new alias: @ffromani

File: SECURITY_CONTACTS [hunk 2]
```diff
@@ -10,9 +10,5 @@
...
 # INSTRUCTIONS AT https://kubernetes.io/security/
 
-cjcullen
-dims
-joelsmith
-liggitt
-philips
-tallclair
+swatisehgal
+fromanirh
```
Comment (ffromani) [inline SECURITY_CONTACTS]: please use the new alias: @ffromani

**Review by ffromani — COMMENTED** [review]
thanks! we just need to update the alias (and perhaps the muscle memory :) )

File: OWNERS (hunk 1 above)
Comment (swatisehgal) [reply OWNERS]: done, Thanks!

File: SECURITY_CONTACTS (hunk 2 above)
Comment (swatisehgal) [reply SECURITY_CONTACTS]: done, Thanks!

File: OWNERS [hunk 3]
```diff
@@ -1,16 +1,11 @@
...
 
 approvers:
-- dims
-- derekwaynecarr
-- dchen1107
-- Huang-Wei
-- sig-node-api-approvers
-- api-approvers
+  - swatisehgal
+  - @ffromani
```
Comment (ffromani) [inline OWNERS]: can we please omit the leading `@` please? (here and everywhere else)

**Review by ffromani — COMMENTED** [review]
sorry to nit again!

File: OWNERS (hunk 3 above)
Comment (swatisehgal) [reply OWNERS]: Ah, right!

Comment (swatisehgal) [reply OWNERS]: Done, Thanks!

**Review by ffromani — APPROVED** [review]
LGTM
thanks for the docs and for the updates

**Pull request merged**